### PR TITLE
feat: add s3 command

### PR DESCRIPTION
**Pull request by rrijkse**
Adds two commands, s3-sync and s3-cp. These are the same as the asw cli and just add the automated part to obtain AWS credentials automatically.

### CODE REVIEW

File: src/commands/s3-sync.yml [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    type: env_var_name
+    description: Vault AppRole Secret ID
+    default: VAULT_SECRET_ID
+  vault-version:
+    type: string
+    description: Install a specific version of vault.
+    default: latest
+
+steps:
+  - ecr-login:
```
Comment (infradannii) [inline src/commands/s3-sync.yml]: Is ecr login required here? - why would the s3 sync need this?

Comment (rrijkse) [reply src/commands/s3-sync.yml]: Sorry I had updated that, then Atom crashed and I didn't notice it hadn't saved that change.

**Review by infradannii — APPROVED** [review]
Looks great! Really excited for this addition 🚀

**Pull request merged**